collector/rapl_linux.go

```diff
@@ -61,7 +62,7 @@ func (c *raplCollector) Update(ch chan<- prometheus.Metric) error {
 		index := strconv.Itoa(rz.Index)
 
 		descriptor := prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "rapl", rz.Name+"_joules_total"),
+			prometheus.BuildFQName(namespace, "rapl", strings.ReplaceAll(rz.Name, "-", "_")+"_joules_total"),
```

Comment:
We should use github.com/prometheus/prometheus/util/strutil

strutil.SanitizeLabelName(k)

Should we move that function to common/ or client_golang/ ?

cc @SuperQ @brian-brazil

Reply:
There's a few of those lying around, it'd be good to gone one in client_golang.

Reply:
Sure.